src/OrleansSQLUtils/Properties/AssemblyInfo.cs

```diff
@@ -21,4 +22,7 @@
 [assembly: InternalsVisibleTo("UnitTests")]
 [assembly: InternalsVisibleTo("TesterInternal")]
 [assembly: InternalsVisibleTo("UnitTestGrains")]
+[assembly: InternalsVisibleTo("Orleans.NonSiloTests")]
```

Comment:
I added to compile. Provider's test still require internals at some point. We discussed that we need to create a "Provider Test Kit" package so we can have it all removed. For now we need keep it.